src/components/Table/Table.less

```diff
@@ -0,0 +1,289 @@
+	// *
+	&-isActive {
+		background-color: @color-primaryMedium;
+		td{
+			&:first-of-type {
+				.hardstop-gradient(to right, @color-primary, @color-primaryMedium, @Table-hover-borderWidth);
+			}
+		}
+		.rowHover(multiply(@color-primaryMedium, @color-lightGray));
+	}
+
+	// **
+	// * A standard `<td>` or `<th>` tag.
+	// *
+	&-cell {
+		padding: 0 @Table-cell-padding;
+		line-height: @Table-row-height;
+		box-sizing: border-box;
+	}
+
+	// **
+	// * Apply to align the text to the left.
+	// *
+	&-align-left {
+		text-align: left;
+	}
+
+	// **
+	// * Apply to align the text to the right.
+	// *
+	&-align-right {
+		text-align: right;
+	}
+
+	// **
+	// * Apply to align the text in the center of a `&-cell`.
+	// *
+	&-align-center {
+		text-align: center;
+	}
+
+	// **
+	// * Add `&-hasCheckbox` if the `&-cell` has a `<Checkbox>` as a child.  This
+	// * will change the width to fit only the `<Checkbox`> and change the
+	// * line-height to inherit to allow the `<Checkbox>` to be centered within
+	// * the cell.
+	// *
+	&-hasCheckbox {
+		width: @Table-hasCheckbox-width;
+		padding: 0 @Table-cell-padding;
+		line-height: inherit;
+	}
+	// **
+	// * Add `&-hasIcon` if the `&-cell` has a `<Icon>` as a child.  This will
+	// * change the width to fit only the `<Icon`> and change the line-height to
+	// * inherit to allow the `<Icon>` to be centered within the cell.
+	// *
+	&-hasIcon {
+		line-height: inherit;
+		width: 37px;
+	}
+
+	// **
+	// * Add `&-hasButton` if the `&-cell` has a `<Button>` as a child. This class
+	// * will center the `<Button>` within the cell.
+	// *
+	&-hasButton {
+		text-align: center;
+	}
+
+	// **
+	// * Add `&-hasBorderLeft` if the `&-cell` needs to have a vertical border
+	// * line on the left.
+	// *
+	&-hasBorderLeft {
+		border-left: @Table-cell-border;
+	}
+
+	// **
+	// * Add `&-hasBorderRight` if the `&-cell` needs to have a vertical border
+	// * line on the right.
+	// *
+	&-hasBorderRight {
+		border-right: @Table-cell-border;
+	}
+
+	&-hasRowSpan {
+		background: @color-white !important;
+	}
+
+	// **
+	// * Add `&-isComfortable` increased the padding/line-height of the cells of
+	// * the the table by 50%. The increased line-heightis applyed only the
+	// * `&-tbody` cells.
+	// *
+	&-isComfortable {
+		th.lucid-Table-cell {
+			&.lucid-Table-hasCheckbox {
+				width: @Table-hasCheckbox-width;
+				line-height: inherit;
+			}
+		}
+		td.lucid-Table-cell {
+			line-height: @Table-row-heightLarge;
+			&.lucid-Table-hasCheckbox {
+				line-height: inherit;
+			}
+			&.lucid-Table-hasIcon {
+				line-height: inherit;
+			}
+		}
+
+	}
+}
+
+// ** MIXIN
+// * This mixin is used to help with the hover state.  The designs call for a
+// * 4px line on the left of the "row".  To do this we use our `.hardstop-gradient`
+// * which is appied to the first `.lucid-Table-cell` as a background gradient.
+// * The mixin takes a single background color that is used to fill the "row"
+// *
+.rowHover(@bgcolor) {
```

Comment:
It's pretty specific to `Table`. I'd rename it `@Table-row-hover` and keep it in this file.